### PR TITLE
yenidonem.com.tr

### PR DESCRIPTION
**Pull request by gork7777**
issue: #103093

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -45,6 +45,7 @@ megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,maciz
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,hdfilmizledayi.com,fullhdfilmizlebaba.com###adStarter
 !
+yenidonem.com.tr##[id^="adContainer"]
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:48]: why without div?

**Comment by Sergey-Lyapin** [comment]
Please also add this rule to annoyances > popups and social > popups
`yenidonem.com.tr##.floating-wpp`

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (gork7777) [reply TurkishFilter/sections/specific.txt:48]: Oh I forgot this

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -114,6 +114,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+yenidonem.com.tr##.floating-wpp
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:117]: @gork7777 and to social > popups

**Pull request merged**